example/newrepo/main.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2018 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// The newrepo command utilizes go-github as a cli tool for
+// creating new repositories.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
+)
+
+func main() {
+	token := os.Getenv("GITHUB_AUTH_TOKEN")
+	name := strings.Join(os.Args[1:], "-")
+	if token == "" {
+		fmt.Println("Unauthorized: No token present")
```

Comment:
Why not use `log.Fatal` here and get rid of the `return` on the next line?
In addition to being more concise, another nice feature is that it will return an error code that could be tested in shell scripts or other calling programs.